provider/terraform/property_override.rb

```diff
@@ -26,6 +26,10 @@ module OverrideFields
       attr_reader :sensitive # Adds `Sensitive: true` to the schema
       attr_reader :validation # Adds a ValidateFunc to the schema
 
+      attr_reader :is_set # Uses a Set instead of an Array
+      # function to determine the unique ID of an item in the set
+      attr_reader :set_hash_func
```

Comment:
Since `set_hash_func` has to be set for every set (and can't be set for any non-set), could we just use `is_set = !set_hash_func.nil?`?

Reply:
Aha, I see it doesn't have to be set for every Set.  I'm okay with having two variables if you add extra docs which show that you don't need to set `set_hash_func` for every Set.

Reply:
Done